recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   sha256: 3c95a732a077c540d1cc58c4c10d6779f0260c8aa63a6e34f79d55689f09ae26
```

Comment:
Looks like [the jobs on `main`]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=625455&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642 ) are now failing due to the checksum changing. This can happen since GitHub archives are generated on-demand as opposed to stored. Thus if a timestamp changes on a file that goes into the archive, the checksum could potentially change.

```python
  File "/opt/conda/lib/python3.10/site-packages/conda_build/source.py", line 109, in download_to_cache
    raise RuntimeError("%s mismatch: '%s' != '%s'" %
RuntimeError: SHA256 mismatch: 'ef54ada5d607387f59950cfb3c3852e5816b62824402d289227b8066e8353489' != '3c95a732a077c540d1cc58c4c10d6779f0260c8aa63a6e34f79d55689f09ae26'
```

Reply:
Would propose we verify we get the same checksum when downloading locally. If so, send a PR to refresh the checksum and bump the `build/number`

Reply:
Submitted PR ( https://github.com/conda-forge/cuda-python-feedstock/pull/29 )